src/pages/index.tsx

```diff
@@ -1,34 +1,56 @@
-import React, { FC } from 'react';
-import { useTranslation, useAppLoaded, Trans } from '@wix/yoshi-flow-bm';
-import { Page, Layout, Cell, Card, Text } from 'wix-style-react';
-
-const introUrl = 'https://github.com/wix-private/business-manager';
+import React, { FC, useState } from 'react';
+import { useAppLoaded, useRequest } from '@wix/yoshi-flow-bm';
+import { Page, Layout, Cell, Card, Text, Input, Button } from 'wix-style-react';
+import { fetchComments, addComments } from '../api/comments.api';
 
 const Index: FC = () => {
   useAppLoaded({ auto: true });
+  const { loading, error, data } = useRequest(fetchComments()); // Not working
```

Comment:
what happened when you try to console log data? error? loading?
which response do you get?
Have you checked the network?